src/test/java/org/broadinstitute/hellbender/tools/spark/BaseRecalibratorSparkIntegrationTest.java

```diff
@@ -148,21 +152,22 @@ public void testBQSRBucket(BQSRTest params) throws IOException {
     public void testPlottingWorkflow() throws IOException {
```

Comment:
This test is disabled -- why can't we re-enable it now as part of this update to the BQSR tests?